BLSQ/iaso
# Polio-1070: global lqas map endpoint

Explain what problem this PR is resolving

Related JIRA tickets : POLIO-1070, POLIO-1001

## Self proofreading checklist


## Changes

- Add `LQASIMGlobalMapViewSet` to get the shapes of each country, calculate its LQAS status and return all that with the country id, so  it can be consumed as is by the front-end
- Update existing front-end code to use endpoint

I overrode the `list` method of the viewset to be able to reuse some code from `org_units.py` and return the shapes.
There's no testing yet as I expect I flouted several good practices and will have to rewrite some stuff.
I made a PR aleady, so it can be quickly reviexed and modified before too much front-end has calcified around the current implementation. 

The PR merges into the dev branch for the polio afro map, so no brutal damage can be done

## How to test

- You will need datastore endpoint, so run `copy_data_store.py` if you haven't yet any in your DB
- Then just go to LQAS>Map, or check the payload at `/api/polio/lqasmap/global/?category=lqas&format=json`

## Print screen / video

Upload here print screens or videos showing the changes

## Notes

@olethanh I don't know if I should put user restrictions (like by location)?
